src/test/java/com/bieganski/ox/BoardTest.java

```diff
@@ -0,0 +1,25 @@
+package com.bieganski.ox;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+@Test
+public class BoardTest {
```

Comment:
You should also test for situations where someone tries to create board or add values to the board outside it's boundries (aka. argument is -1 or 100000000000)